portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -77,7 +77,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String userLastName = getParameter(request, "userLastName", /* defaultValue= */ "");
     String userMessage = getParameter(request, "userMessage", /* defaultValue= */ "");
 
-    // TODO(lornaarmstrong) Fix this issue - 'blank' comments are still posted
     if ( (!(userMessage.isEmpty())) && (!(userFirstName.isEmpty())) &&(!(userLastName.isEmpty()))) {
```

Comment:
Nit: Just sharing a good practice which we follow and I believe produces a bit more readable code :) We try to write such cases like the following

```
if (something_is_invalid) {
  // throw an exception or simply return
}
// do the tasks you should do if conditions are valid
```